js/glsl/scatter-vertex.glsl

```diff
@@ -50,7 +50,31 @@ attribute vec3 color_previous;
 void main(void) {
     vec3 origin = vec3(xlim.x, ylim.x, zlim.x);
     vec3 size_viewport = vec3(xlim.y, ylim.y, zlim.y) - origin;
+#ifdef USE_SPRITE
+    vec3 vector = v;
+    vec3 vector_previous = v_previous;
+    vec3 position_offset = vec3(x, y, z);
+    vec3 position_offset_previous = vec3(x_previous, y_previous, z_previous);
 
+    float s = mix(size_previous/100., size/100., animation_time_size);
+    vec3 pos = (mix(position_offset_previous, position_offset, vec3(animation_time_x, animation_time_y, animation_time_z))
+                - origin) / size_viewport - 0.5;
+    vec4 posxy = projectionMatrix *
+                modelViewMatrix *
+                vec4(pos,1.0);
+    // Test vector used to correct scale in clipspace
+    vec4 posxy_test = projectionMatrix *
+                modelViewMatrix *
+                vec4(1.0,0.0,0.0,1.0);
+    float posxy_mag = sqrt(posxy_test.x*posxy_test.x +
```

Comment:
what does this do? I noticed when rotating example/0001.html that the size of the sprite changes, what is the idea behind it?

Reply:
Good question. At the moment `size` is a percentage of the viewport -- which I found very useful. Without this code, `size` refers to a number of pixels... but the size of the sprites shouldn't change, which means I worked in size attenuation unintentionally... I'll rethink this. 

Unless you're happy with `size` meaning a number of pixels in the case of sprites.

Reply:
no, it would be nice if it is consistent. It seems right actually.